docs/src/user-guide/url-parameters.md

```diff
@@ -0,0 +1,65 @@
+# URL Parameters
+
+The YScope Log Viewer supports various URL parameters to control its behavior and state. The 
+parameters are divided into two categories:
+
+- **Search parameters** (after `?`).
+- **Hash parameters** (after `#`).
+
+## Search Parameters
+
+Search parameters appear after the `?` in the URL, and manual modification in the browser address
+bar causes the page to reload. Such parameters are used to initialize the log viewer.
+
+| Parameter | Type | Default | Description | Example                                         |
+|-----------|------|---------|-------------|-------------------------------------------------|
+| `filePath` | String | "" | Specifies the log file to load on startup | `?filePath=https://example.com/app-log.clp.zst` |
+
+**Notes for filePath:**
+- Can be a local file URL or remote HTTP(S) URL.
+- If the file URL is not [percent-encoded][rfc-3986-percent-encoding] as per RFC 3986, it must be
```

Comment:
percent-decoding should work but currently doesn't. it should by fixed by #151

Reply:
actually this should have been fixed by #295 but was regressed in #286 

@coderabbitai create an issue to track this